krun.py

```diff
@@ -277,9 +307,32 @@ def main():
     config = util.read_config(config_file)
     out_file = util.output_name(config_file)
 
+    mail_recipients = config.get("MAIL_TO", [])
+    if type(mail_recipients) is not list:
+        fatal("MAIL_TO config should be a list")
+
+    max_mails = config.get("MAX_MAILS", 5)
+
+    # You should only have mailing for real experiments.
+    # Prevent accidental sending of mail during benchmark implementation
```

Comment:
If you want to add a "don't send email switch" (-n or whatever), that would be better I think. But I don't think that's even necessary. At worst people are going to get 5 mails... and if they don't want any, they can simply not specify a mail address.